src/main/java/org/folio/marc4ld/service/ld2marc/resource/field/ControlFieldsBuilder.java

```diff
@@ -14,6 +15,10 @@ public class ControlFieldsBuilder {
   private static final String EMPTY_CF = SPACE.repeat(39);
   private final Map<String, StringBuilder> storage = new LinkedHashMap<>();
```

Comment:
minor
move init to constructor if its present

Reply:
It's not even minor, it's controversial and matter of taste :)